ironfish/src/rpc/adapters/socketAdapter/socketAdapter.ts

```diff
@@ -80,6 +80,12 @@ export abstract class RpcSocketAdapter implements IRpcAdapter {
     this.inboundTraffic.start()
     this.outboundTraffic.start()
 
+    if (this.listen.path) {
+      await fsAsync.unlink(this.listen.path).catch(() => {
```

Comment:
Yea interesting. I wonder if we'd ever run into problems if a node was started while another node is already running. Right it doesn't get past the DB in use error so we don't have any issues.